src/Hashing.js

```diff
@@ -0,0 +1,29 @@
+/* global require exports BROWSER_RUNTIME */
+
+const Blake2 = require('blakejs');
```

Comment:
I wonder if we could use `blake2.wasm` and `node-blake2` packages for speed, since we have conditional compilation anyway.

This is not a change request, performance is probably acceptable here.